src/services/WolfDen.Application/Requests/Queries/Attendence/DailyAttendance/DailyDetailsQueryHandler.cs

```diff
@@ -38,38 +39,47 @@ public async Task<DailyAttendanceDTO> Handle(DailyDetails request, CancellationT
                     }
                     else
                     {
-                        LeaveRequest leave = await _context.LeaveRequests.Where(x => x.EmployeeId == request.EmployeeId && x.FromDate == request.Date && x.LeaveRequestStatusId == LeaveRequestStatus.Approved).Include(x => x.LeaveType).FirstOrDefaultAsync(cancellationToken);
+                        LeaveRequest leave = await _context.LeaveRequests.Where(x => x.EmployeeId == request.EmployeeId && x.FromDate <= request.Date && request.Date <= x.ToDate && x.LeaveRequestStatusId == LeaveRequestStatus.Approved).Include(x => x.LeaveType).FirstOrDefaultAsync(cancellationToken);
                         if (leave is null)
                         {
                             AttendanceStatus attendanceStatusId = AttendanceStatus.Absent;
                             notPresentDay.AttendanceStatusId = attendanceStatusId;
                         }
                         else
                         {
-                            LeaveType leaveType = await _context.LeaveType.FirstOrDefaultAsync(x => x.Id == leave.TypeId);
-                            if (leaveType.LeaveCategoryId == LeaveCategory.WorkFromHome)
-                            {
-                                AttendanceStatus attendanceStatusId = AttendanceStatus.WFH;
-                                notPresentDay.AttendanceStatusId = attendanceStatusId;
-                            }
-                            else
-                            {
-                                AttendanceStatus attendanceStatusId = AttendanceStatus.RestrictedHoliday;
-                                notPresentDay.AttendanceStatusId = attendanceStatusId;
-                            }
+                            AttendanceStatus attendanceStatusId = AttendanceStatus.RestrictedHoliday;

```

Comment:
cant we just assign notPresentDay.AttendanceStatusId = AttendanceStatus.RestrictedHoliday;